modules/web/src/app/cluster/details/cluster/rbac/dialog/add-service-account/theme.scss

```
@@ -0,0 +1,27 @@
+// Copyright 2020 The Kubermatic Kubernetes Platform contributors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+@use 'sass:map';
+
+@mixin theme-add-binding-component($colors) {
```

```suggestion
@mixin theme-add-service-account-component($colors) {
```
This needs to be registered in `src/assets/css/theme/_component.scss` as well.